sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -131,6 +131,23 @@ def begin_translation(
             source URL to documents and can contain multiple TranslationTargets (one for each language)
             for the destination to write translated documents.
         :type inputs: List[~azure.ai.translation.document.DocumentTranslationInput]
+        :keyword str source_language_code: Language code for the source documents.
+            If none is specified, the source language will be auto-detected for each document.
+        :keyword str prefix: A case-sensitive prefix string to filter documents in the source path for
+            translation. For example, when using a Azure storage blob Uri, use the prefix to restrict
+            sub folders for translation.
+        :keyword str suffix: A case-sensitive suffix string to filter documents in the source path for
+            translation. This is most often use for file extensions.
+        :keyword storage_type: Storage type of the input documents source string. Possible values
+            include: "Folder", "File".
+        :paramtype storage_type: str or ~azure.ai.translation.document.StorageInputType
+        :keyword str category_id: Category / custom model ID for using custom translation.
+        :keyword glossaries: Glossaries to apply to translation.
+        :paramtype glossaries: list[~azure.ai.translation.document.TranslationGlossary]
+        :keyword str source_storage_source: Source storage source. Default value: "AzureBlob".
```

Comment:
Slightly weird how you have "Default value is "AzureBlob". Only "AzureBlob" supported"

I think for now you can just have the "Only AzureBlob supported" for now, and once more values get added, switch it to "Default value is AzureBlob"

Reply:
curious, why is Python exposing this property if the user can't set other values right now?
in .NET we decided to hide it, and whenever there are other options we will expose it.

Reply:
I was on the fence about it and Johan didn't have a preference for exposing it. Seeing as .NET is hiding it, I will hide as well. Thanks!